srcpkgs/ClusterSSH/patches/15config.diff

```diff
@@ -0,0 +1,28 @@
+--- a/t/15config.t
```

Comment:
can you add a comment at the top saying this test fails in ci because it doesn't expect root?

Reply:
Alright

Reply:
Pushed